types/src/validator_set.rs

```diff
@@ -36,6 +37,21 @@ pub(crate) fn validator_set_path() -> Vec<u8> {
 #[cfg_attr(any(test, feature = "testing"), derive(Arbitrary))]
 pub struct ValidatorSet(Vec<ValidatorPublicKeys>);
 
+impl fmt::Display for ValidatorSet {
+    fn fmt(&self, f: &mut fmt::Formatter) -> std::fmt::Result {
+        // I still don't understand why Rust doesn't want me to be able to display vectors,
+        // but whatever.
+        write!(
+            f,
+            "[{}]",
+            self.0.iter().fold(String::new(), |acc, validator| format!(
```

Comment:
I think this is simpler `self.0.map(into_string).join(", ")`

Reply:
Oh yea, let me try! It looks simpler indeed :)